cmd/version.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"strings"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+//Default version if tag not available
+var (
+	Version string = "Not Available"
+)
+
+//Getting the latest GitHub Version from tag
+func getVersion() string {
+	cmd := exec.Command("git", "describe", "--abbrev=0", "--tags")
```

Comment:
It implies to have `git` locally, can we think about making an http call to github API? https://developer.github.com/v3/git/tags/